tests/aws/services/sns/test_sns.py

```diff
@@ -765,10 +768,23 @@ def test_subscribe_idempotency(
         )
         snapshot.match("subscribe-idempotent", subscribe_resp)
 
-        get_attrs_resp = aws_client.sns.get_subscription_attributes(
-            SubscriptionArn=subscribe_resp["SubscriptionArn"]
```

Comment:
q: why is this deleted now? I guess this check was there to verify that the subscription wasn't updated. We could do it also after subscribing with empty attributes, but I suppose the subscription stays the same